pkgs/applications/window-managers/wayfire/wf-shell.nix

```diff
@@ -15,7 +16,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ meson ninja pkg-config wayland ];
   buildInputs = [
-    alsa-lib gtkmm3 gtk-layer-shell pulseaudio wayfire wf-config
+    alsa-lib gtkmm3 gtk-layer-shell pulseaudio wayfire wf-config wlroots
```

Comment:
should `wlroots` be in propagatedBuildInputs for `wayfire`?

Reply:
@Pacman99 I don't see an obvious reason why it should be in `propagatedBuildInputs`, why would you propose that?